src/test/java/com/uid2/shared/secure/azurecc/MaaTokenSignatureValidatorTest.java

```diff
@@ -22,7 +22,28 @@ public void testPayload() throws Exception {
         var expectedLocation = "East US";
         var expectedPublicKey = "abc";
 
-        var tokenPayload = validateAndParseToken(payload, clock);
```

Comment:
should testPayload() be renamed  like testAciPayload() to make clearer the diff?